classes/item.rb

```diff
@@ -0,0 +1,45 @@
+# item.rb
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :archived
+  attr_reader :id, :genre, :author, :label, :source
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1...1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def genre=(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  def author=(author)
+    @author = author
+    author.items << self unless author.items.include?(self)
+  end
+
+  def label=(label)
+    @label = label
+    label.items.push(self) unless label.items.include?(self)
+  end
+
+  def source=(source)
+    @source = source
+    source.items.push(self) unless source.items.include?(self)
+  end
+
```

Comment:
- [ ] You don't need to setup this association (1-to-many relationships) for `source` as this is for only a group with team members of 4. You are only a two persons team and therefore this is not required. :+1: :wave: